examples/client_replication/src/client.rs

```diff
@@ -132,14 +135,19 @@ fn player_movement(
     }
 }
 
-/// Spawn a client-owned player entity when the space command is pressed
+/// Spawn a server-owned pre-predicted player entity when the space command is pressed
 fn spawn_player(
     mut commands: Commands,
     mut input_reader: EventReader<InputEvent<Inputs>>,
     connection: Res<ClientConnection>,
     players: Query<&PlayerId, With<PlayerPosition>>,
+    config: Res<ClientConfig>,
 ) {
     let client_id = connection.id();
+    if config.shared.mode == Mode::HostServer && !client_id.is_local() {
```

Comment:
Can this be replaced by some sort of `has_authority` run condition like in replicon?

Reply:
As mentioned above, this is fixed by separating the ConnectEvents.

I do think that it will be necessary to have more control in lightyear over who has authority. I see that unreal has more finegrained control over 'ControlledPawns', etc.
It's definitely something I want to look into